### PR TITLE
vote: update(OWNERS): add new maintainers and reviewers

### LINKED ISSUE #136: Revamping falcoctl: call for core maintainers
Dear @falcosecurity/core-maintainers  :wave: 

As discussed recently, we have the intention of revamping this project. From what I understood until now, we have a short-term goal and a long-term dream.

In the short term, the idea is to make `falcoctl` the tool for installing plugins and rules artifacts (see https://github.com/falcosecurity/falcoctl/issues/103). Beyond that, I dream that this tool will replace the `falco-driver-loader` bash script in the future. 

## Short-term proposal

However, this ambitious plan needs your help. Unfortunately, I'm the only active maintainer of this repo, and as per our governance, we need at least two maintainers to make crucial decisions. For this reason, as a first step, I want to invite any willing core maintainer to step up (even temporarily) to become a maintainer of this repo ASAP.
(why core maintainers? see [this](https://github.com/falcosecurity/evolution/blob/main/REPOSITORIES.md#core-maintainers-duties-and-privileges)).  

**If you're interested, please comment** below. We will open a PR to vote on that once we have some volunteers.

Also, if nobody disagrees, I want to sponsor @alacuku and @loresuso as reviewers since they are doing an egregious job reworking this project and implementing the plugin/rules manager functionality.

## Next steps

The solution above will unblock the repository from a stagnant situation, but it's not enough for the project's future. Once the repo is functioning again, we should spend our effort seeking new contributors and maintainers (possibly increasing the maintainers' diversity).

One possible solution would be to start a structured mentoring program as described [here](https://github.com/falcosecurity/evolution/blob/main/MAINTAINERS_GUIDELINES.md#mentoring-programs).

### PR DESCRIPTION
**Pull request by jasondellaluce**
**What type of PR is this?**

/kind documentation

**Any specific area of the project related to this PR?**

Following up the discussion in https://github.com/falcosecurity/falcoctl/issues/136 and in the last community call, this PR **opens the voting** for the addition of new approvers and reviewers (although we would not need a formal vote for reviewers).

For this kind of decision, a simple majority vote among core maintainers (cc @falcosecurity/core-maintainers) is enough to proceed with the changes (see: https://github.com/falcosecurity/evolution/blob/main/GOVERNANCE.md#majority-vote). Whoever is subject to the changes proposal is not eligible for voting (@zuc, @maxgio92, @cpanato, @FedeDP).

**Voting deadline**: 2022-09-15

Close #136

### CODE REVIEW

**Comment by jasondellaluce** [comment]
/hold

Until the voting is concluded.

File: OWNERS [hunk 1]
```diff
@@ -1,7 +1,13 @@
 approvers:
   - leogr
+  - zuc
+  - maxgio92
+  - fededp
+  - cpanato
 reviewers:
   - mfdii
```
Comment (leogr) [inline OWNERS:8]:
```suggestion
```

We should remove `mfdii` since is inactive for more than a year.

Comment (jasondellaluce) [reply OWNERS:8]: Agreed.

**Comment by leogr** [comment]
Ofc +1 from me :rocket:

**Comment by jasondellaluce** [comment]
+1

I love the initiative of revamping this project and I'm looking forward to seeing new great things from this new leadership group.

**Comment by Andreagit97** [comment]
+1

Can't wait to see what we can do with this tool :heart_eyes:

**Comment by cpanato** [comment]
+1

**Comment by leogr** [comment]
We are close to declaring this proposal successful :partying_face: 

Meanwhile, @loresuso could you open a PR to add yourself as an organization member :point_down: ?
https://github.com/falcosecurity/test-infra/blob/master/config/org.yaml#L20

**Comment by loresuso** [comment]
@leogr Yes of course! I am opening the PR 🙂

**Review by leogr — APPROVED** [review]
Close #136 

:partying_face:

**Pull request merged**